### DISCUSSION TITLE
SFX incorporated into themes

### DISCUSSION THREAD

**Discussion by SundaePixel** [godotengine/godot-proposals#5816] [GUI]
Hey! I want to be able to play a certain SFX whenever hovering, or clicking a button without manually having to do so with every single button via scripts. That would be very good but sadly I can't find a very scalable way to do it with all the buttons i have in my game. So yes, I am here to propose this idea and I think it would be very useful for plenty others as well! I was thinking it could be incorporated into themes. Thank you very much for reading and share your opinions! 👍

**Comment by Calinou** [comment]
This is already being tracked in https://github.com/godotengine/godot-proposals/issues/1472.
